docs/tutorials/text-displayer.md

```diff
@@ -1,15 +1,38 @@
 # Configuring text displayer
 
-Shaka Player supports two types of Text Displayer.
-1) Using the browser's native API, Shaka Player creates a textTrack along
-   with the AUDIO or VIDEO Element.
-2) Rendering the subtitles in a container in the DOM.
-3) Providing a custom text displayer factory would allow a developer to make
-   their own custom text displayer that fits neither category.
-
-By default, if not configured otherwise, the player will use type 1.
-To configure type 2, you have to call `setVideoContainer` function before
-making `attach` call in the player.
-
-Note: The UI calls setVideoContainer for you if the video's controls are
-disabled, so this isn't always necessary when using the UI.
+### Default displayers
+
+Shaka Player supports two implementations of {@link shaka.extern.TextDisplayer}.
+1) {@link shaka.text.SimpleTextDisplayer} which uses browser's native cue
+  renderer. Shaka Player creates a custom text track attached to the video
+  element and provides necessary data so video element can render it. This is
+  the default displayer when shaka UI is **not** used.
+2) {@link shaka.text.UITextDisplayer} which renders subtitles inside of a DOM
+  container provided to shaka by a {@link shaka.Player#setVideoContainer}
+  method. This call is done automatically when using Shaka UI.
```

Comment:
you should also mention that it's possible configure it in the constructor
https://github.com/shaka-project/shaka-player/blob/main/lib/player.js#L599

Reply:
Done!